### PR TITLE
Update 3

### PR DESCRIPTION
**Pull request by lftan**
- some chapter updates
- Fix unable display images in *.adoc issue

### CODE REVIEW

File: src/srvgrp-hart-state-management.adoc [hunk 1]
```diff
@@ -21,14 +22,14 @@ Below table lists the services in this group:
...
 ==== Service: *HSM_ENABLE_NOTIFICATION*
-This service allows AP to subscribe to system reset service group notifications.
-Platform can optionally support notifications of events which might occur in the
-platform. PuC can send these notification messages to AP if they are implemented
-and AP has subscribed to these. Events supported are described in
-<<#hsm-notifications>>.
+This service allows the AP to subscribe to HART_STATE_MANAGEMENT service group
+notifications. The platform can optionally support notifications of events may
+occur in the platform. PuC can send these notification messages to the AP if
+they are implemented and the AP has subscribed to them. Supported event are
```
Comment (pathakraul) [inline src/srvgrp-hart-state-management.adoc:41]: %s/event/events

File: src/srvgrp-hart-state-management.adoc [hunk 2]
```diff
@@ -50,27 +51,28 @@ notification.
...
 |===
 
 ==== Service: *HSM_HART_START*
-This service helps start (or power up) a hart with a specified hart-id. 
-Successful completion of this service means that hart with specified hart-id has
-started execution from the provided start address. The previous state of the 
-hart before this service was called is platform specific. It's possible that 
-hart was already started or hart with specified hart-id does not exist. 
-Implementation should return proper error code in the status field accordingly.
+This service helps to start (or power up) a hart with a specified `HART_ID`.
```
Comment (pathakraul) [inline src/srvgrp-hart-state-management.adoc:70]: better to write "This service is used to start" instead of help

Comment (lftan) [reply src/srvgrp-hart-state-management.adoc:70]: ok

**Comment by lftan** [comment]
Updated.

**Pull request merged**